syncserver/src/web/extractors.rs

```diff
@@ -1173,7 +1170,10 @@ impl FromRequest for HawkIdentifier {
                 "tokenserver_origin",
                 &hawk_id.tokenserver_origin.to_string(),
             );
-            tags.commit(&mut exts);
```

Comment:
I removed this method from the `Tags` type because it depended on Actix Web's request extensions. Now that the `Tags` lives in `syncserver-common` (which has no other dependency on Actix), I decided to remove it. It's only called in two places anyway